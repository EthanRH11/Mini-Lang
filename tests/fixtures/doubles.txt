begin:
    double pi = 3.14;

    double new = 7.87;

    double add = pi + new;

    out_to_console(pi); ...
    out_to_console(new); ...
    out_to_console(add); ...

end