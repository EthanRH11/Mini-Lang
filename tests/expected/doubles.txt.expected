3.14
7.87
11.01